docs/api/rate-limits.md

```diff
@@ -51,3 +51,9 @@ The following tables list the rate limits for the NuGet.org API.
 |:---|:---|:---|:--- |
 **PUT** `/api/v2/package` | API Key | 350 / hour | Upload a new NuGet package (version) via v2 push endpoint 
 **DELETE** `/api/v2/package/{id}/{version}` | API Key | 250 / hour | Unlist a NuGet package (version) via v2 endpoint 
+
+## NuGet Gallery page views
+
+| API | Limit Type | Limit Value | API usecase | 
+|:---|:---|:---|:--- |
+**GET** `/package/{id}/{version}` | IP | 50 / minute | Display package (version) details page. 
```

Comment:
Maybe say something like:

`
If you are accessing the NuGet Gallery web pages programmatically, consider investigating our documented [V3 API](relative-path-here). These endpoints allow for simpler access to package metadata and content. The V3 API has better availability and has higher performance than accessing the NuGet Gallery web pages, which are designed for web browser interactio.
`

.. or something like that.

Reply:
Are people aware that "NuGet Gallery" means the nuget.org website? Can we just say nuget.org website instead?